sql-jdbc/build.gradle

```diff
@@ -120,47 +117,59 @@ publishing {
             artifact javadocJar
 
             pom {
-              name = "OpenSearch SQL JDBC Driver"
-              packaging = "jar"
-              url = "https://github.com/opensearch-project/sql"
-              description = "OpenSearch SQL JDBC driver"
-              scm {
-                connection = "scm:git@github.com:opensearch-project/sql.git"
-                developerConnection = "scm:git@github.com:opensearch-project/sql.git"
-                url = "git@github.com:opensearch-project/sql.git"
-              }
-              licenses {
-                license {
-                  name = "The Apache License, Version 2.0"
-                  url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                name = "OpenSearch SQL JDBC Driver"
+                packaging = "jar"
+                url = "https://github.com/opensearch-project/sql/sql-jdbc"
+                description = "OpenSearch SQL JDBC driver"
+                scm {
+                    connection = "scm:git@github.com:opensearch-project/sql.git"
+                    developerConnection = "scm:git@github.com:opensearch-project/sql.git"
+                    url = "git@github.com:opensearch-project/sql.git"
+                }
+                licenses {
+                    license {
+                        name = "The Apache License, Version 2.0"
+                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
                 }
-              }
-              developers {
-                developer {
-                  id = "amazonwebservices"
-                  organization = "Amazon Web Services"
-                  organizationUrl = "https://aws.amazon.com"
+                developers {
+                    developer {
+                        name = 'OpenSearch'
+                        url = 'https://github.com/opensearch-project/sql'
+                    }
                 }
-              }
             }
         }
-    }
+        publishMaven(MavenPublication) { publication ->
```

Comment:
What is the group of the jdbc?
`org.opensearch.client.opensearch-sql-jdbc`

Reply:
Currently, `org.opensearch.client`. What should be?

Reply:
Hey @Yury-Fridlyand and @peterzhuamazon should this be `org.opensearch.driver` as groupId? and artifact name should be `opensearch-sql-jdbc`.

Reply:
Fixed in 79deba0b.

Reply:
See my next comment on this.